hw/ip/pinmux/doc/_index.md

```diff
@@ -6,122 +6,383 @@ title: "Pinmux Technical Specification"
 # Overview
 
 This document specifies the functionality of the pin multiplexer (`pinmux`) peripheral.
-This module conforms to the [OpenTitan guideline for peripheral device functionality.]({{< relref "doc/rm/comportability_specification" >}}).
+This module conforms to the [OpenTitan guideline for peripheral device functionality]({{< relref "doc/rm/comportability_specification" >}}).
 See that document for integration overview within the broader OpenTitan top level system.
 The module provides a mechanism to reconfigure the peripheral-to-pin mapping at runtime, which greatly enhances the system flexibility.
-This IP is closely related to the `padctrl` instance which provides additional control of pad attributes (pull-up, pull-down, open drain, drive strength, keeper and inversion).
-See  (**TODO**) for more information.
-
-
+In addition to that, the `pinmux` also allows to control pad attributes (such as pull-up, pull-down, open-drain, drive-strength, keeper and inversion) and it contains features that facilitate low-power modes of the system.
```

Comment:
allows to control --> allows you to control

inversion) and --> inversion), and

Reply:
thanks, I've slightly rewritten this to `this allows the user to control...` since `you` sounded a bit odd in this sentence.